{"uid":"40895f08515a6784","name":"test_edit_account_addressShipping_invalid","fullName":"test.Cuentas.test_edit_account#test_edit_account_addressShipping_invalid","historyId":"de6cde3caf654862b0e0a890bdd6ef0e","time":{"start":1726762244096,"stop":1726762244096,"duration":0},"status":"skipped","statusMessage":"XFAIL This test case is expected to fail due to known issue.\n\nTypeError: list indices must be integers or slices, not str","statusTrace":"setup_edit_account = ({'Espo-Authorization': 'YWRtaW46ODAwNDkwNTM=', 'X-Api-Key': '4c29fe9482f3bccb7ac0a7d04015a2e7'}, [{'assignedUserId': None, 'assignedUserName': None, 'billingAddressCity': None, 'billingAddressCountry': None, ...}])\n\n    @pytest.mark.regression\n    @pytest.mark.functional\n    @pytest.mark.xfail(reason=\"This test case is expected to fail due to known issue.\", condition=True)\n    def test_edit_account_addressShipping_invalid(setup_edit_account):\n        headers, account = setup_edit_account\n        payload_account = PayloadAccount().build_payload_add_account(name=\"prueba nueva\", billingAddressCity=\"prueba\",\n                                                                     billingAddressCountry=\"prueba\",\n                                                                     billingAddressPostalCode=\"prueba\",\n                                                                     billingAddressState=\"prueba\",\n                                                                     billingAddressStreet=\"prueba\",\n                                                                     description=\"prueba\", emailAddress=\"prueba@prueba.com\",\n                                                                     industry=\"Architecture\", phoneNumber=\"+15555555555\",\n                                                                     shippingAddressCity=\" \",\n                                                                     shippingAddressCountry=\" \",\n                                                                     shippingAddressPostalCode=\" \",\n                                                                     shippingAddressState=\" \",\n                                                                     shippingAddressStreet=\" \", type=\"Customer\",\n                                                                     website=\"prueba.com\")\n>       response = EspocrmRequest().put(EndpointAccount.view(account['id']), headers, payload_account)\nE       TypeError: list indices must be integers or slices, not str\n\ntest/Cuentas/test_edit_account.py:165: TypeError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"get_headers","time":{"start":1726762227223,"stop":1726762227224,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"setup_edit_account","time":{"start":1726762243930,"stop":1726762244096,"duration":166},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"afterStages":[{"name":"setup_edit_account::0","time":{"start":1726762244100,"stop":1726762244100,"duration":0},"status":"broken","statusMessage":"TypeError: list indices must be integers or slices, not str\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/allure_commons/_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/_pytest/fixtures.py\", line 907, in _teardown_yield_fixture\n    next(it)\n  File \"/home/runner/work/espo-automation-final/espo-automation-final/test/Cuentas/conftest.py\", line 58, in setup_edit_account\n    AccountCall().delete(headers, account['id'])\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true}],"labels":[{"name":"tag","value":"@pytest.mark.xfail(reason='This test case is expected to fail due to known issue.', condition=True)"},{"name":"tag","value":"functional"},{"name":"tag","value":"regression"},{"name":"parentSuite","value":"test.Cuentas"},{"name":"suite","value":"test_edit_account"},{"name":"host","value":"fv-az1116-0"},{"name":"thread","value":"2157-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"test.Cuentas.test_edit_account"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"tags":["functional","regression","@pytest.mark.xfail(reason='This test case is expected to fail due to known issue.', condition=True)"]},"source":"40895f08515a6784.json","parameterValues":[]}